sdk/trace/src/main/scala/org/typelevel/otel4s/sdk/trace/autoconfigure/SpanExportersAutoConfigure.scala

```diff
@@ -0,0 +1,150 @@
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import org.typelevel.otel4s.sdk.autoconfigure.AutoConfigure
+import org.typelevel.otel4s.sdk.autoconfigure.Config
+import org.typelevel.otel4s.sdk.autoconfigure.ConfigurationError
+import org.typelevel.otel4s.sdk.trace.exporter.SpanExporter
+
+/** Autoconfigures [[SpanExporter]]s.
+  *
+  * The configuration options:
+  * {{{
+  * | System property      | Environment variable | Description                                                                                   |
+  * |----------------------|----------------------|-----------------------------------------------------------------------------------------------|
+  * | otel.traces.exporter | OTEL_TRACES_EXPORTER | The exporters to use. Use a comma-separated list for multiple propagators. Default is `otlp`. |
+  * }}}
+  *
+  * @see
+  *   [[https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#propagator]]
+  */
+private final class SpanExportersAutoConfigure[F[_]: MonadThrow: Console](
+    extra: Set[AutoConfigure.Named[F, SpanExporter[F]]]
+) extends AutoConfigure.WithHint[F, Map[String, SpanExporter[F]]](
+      "SpanExporters",
+      SpanExportersAutoConfigure.ConfigKeys.All
+    ) {
+
+  import SpanExportersAutoConfigure.ConfigKeys
+  import SpanExportersAutoConfigure.Const
+
+  private val configurers = {
+    val default: Set[AutoConfigure.Named[F, SpanExporter[F]]] = Set(
+      AutoConfigure.Named.const(Const.NoneExporter, SpanExporter.noop[F])
+    )
+
+    default ++ extra
+  }
+
+  def fromConfig(config: Config): Resource[F, Map[String, SpanExporter[F]]] = {
+    val values = config.getOrElse(ConfigKeys.Exporter, Set.empty[String])
+    Resource.eval(MonadThrow[F].fromEither(values)).flatMap {
+      case names if names.contains(Const.NoneExporter) && names.sizeIs > 1 =>
+        Resource.raiseError(
+          ConfigurationError(
+            s"[${ConfigKeys.Exporter}] contains '${Const.NoneExporter}' along with other exporters"
+          ): Throwable
+        )
+
+      case exporterNames =>
+        val names = NonEmptyList
+          .fromList(exporterNames.toList)
+          .getOrElse(NonEmptyList.one(Const.OtlpExporter))
+
+        names
+          .traverse(name => create(name, config).tupleLeft(name))
+          .map(_.toList.toMap)
+    }
+  }
+
+  private def create(name: String, cfg: Config): Resource[F, SpanExporter[F]] =
+    configurers.find(_.name == name) match {
+      case Some(configure) =>
+        configure.configure(cfg)
+
+      case None =>
+        Resource.eval(otlpMissingWarning.whenA(name == Const.OtlpExporter)) >>
+          Resource.raiseError(
+            ConfigurationError.unrecognized(
+              ConfigKeys.Exporter.name,
+              name,
+              configurers.map(_.name)
+            ): Throwable
+          )
+    }
+
+  private def otlpMissingWarning: F[Unit] = {
+    Console[F].errorln(
```

Comment:
I don't have any better idea how to communicate with the user that dependency is missing. The service will not start anyway, but we will also provide a meaningful help message.